<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QLabel" name="select">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECIONE A FUNÇÃO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_fechar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(165, 165, 165);</string>
   </property>
   <property name="text">
    <string>fechar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="bt_admin">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 165, 165);</string>
      </property>
      <property name="text">
       <string>administrador</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_pedidos">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 165, 165);</string>
      </property>
      <property name="text">
       <string>pedidos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_balcao">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 165, 165);</string>
      </property>
      <property name="text">
       <string>balcão</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_caixa">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 165, 165);</string>
      </property>
      <property name="text">
       <string>caixa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_desen">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 165, 165);</string>
      </property>
      <property name="text">
       <string>desenvolvedor</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
